src/main/java/com/esri/samples/map/create_and_save_map/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Create and Save Map</h1>
+
+<p>How to create and save a map to your own portal.</p>
+
+<p><img src="CreateAndSaveMap.png"/></p>
```

Comment:
When you get to the username/password part does it let you login in? When I try to sign in it says it is signing in but never does.